README.md

```diff
@@ -96,6 +96,24 @@ Format the Cairo source code (using Scarb):
 $ make fmt
 ```
 
+### Step 4: Setup Language Server
+
+#### VS Code Extension
+
+- Disable previous Cairo 0.x extension
+- Install the Cairo 1 extension for proper syntax highlighting and code navigation.
+Just follow the steps indicated [here](https://github.com/starkware-libs/cairo/blob/main/vscode-cairo/README.md).
+
+#### Cairo Language Server
+
+From [Step 1](#step-1-install-cairo-10-guide-by-abdel), the `cairo-language-server` binary should be built and executing this command will output the path, if it doesn't exist it will be built.
+
+```bash
```

Comment:
I like the idea of helping users find the extension path. I think we can remove the Makefile changes and just do this to copy the extension path to clipboard:

```
$ which cairo-language-server | pbcopy
```

Reply:
Nice agree with this 🚀